app/src/main/java/zegreanu/cristi/geoquiz/Question.kt

```diff
@@ -0,0 +1,4 @@
+package zegreanu.cristi.geoquiz
+
+class Question(var textResId: Int, var answerTrue: Boolean) {
```

Comment:
You can probably make this a Data Class (https://kotlinlang.org/docs/reference/data-classes.html)

Reply:
also, the properties in it can be vals (they don't need setters)